core-bundle/src/DataContainer/DataContainerOperationsBuilder.php

```diff
@@ -48,7 +48,7 @@ public function __toString(): string
         ]);
     }
 
-    public function initializeButtons(string $table, array $record, DataContainer $dataContainer, callable|null $legacyCallback = null): self
+    public function initializeEmpty(): self
```

Comment:
```suggestion
    public function initialize(): self
```

And `initializeWithButtons()` instead of `initializeButtons()`. WDYT?

Reply:
Fine with me. Changed in 503d334ffc3e2a3918d62c43c1234409d2af181d 🙃